hasura.planx.uk/migrations/1725611185485_alter_table_public_teams_drop_column_submission_email/down.sql

```diff
@@ -0,0 +1,3 @@
+comment on column "public"."teams"."submission_email" is E'Teams are core way we organise `flows` and configure settings like theme colour, logo, and contact info; name usually reflects a council but not exclusively';
+alter table "public"."teams" alter column "submission_email" drop not null;
+alter table "public"."teams" add column "submission_email" text;
```

Comment:
I don't think this is a valid `down` migration - you can't `comment on` or `alter` a column before it exists/has been added.

I think this should be: 
```sql
alter table "public"."teams" add column "submission_email" text;
comment on column "public"."teams"."submission_email" is E'Teams are core way we organise `flows` and configure settings like theme colour, logo, and contact info; name usually reflects a council but not exclusively';
```